build/vagrant/buildspec-vagrant.yml

```diff
@@ -0,0 +1,16 @@
+---
+version: 0.2
+
+phases:
+  install:
+    commands:
+  build:
+    commands:
+      - cd build/vagrant/ && make
+  post_build:
+    commands:
```

Comment:
not saying it should go in the buildspec or if its even required, but should there be any kind of validation of the vagrant image itself post creation? Or is that built into the `packer build` command?

Reply:
I don't know if packer validates the vagrant box at all.

Reply:
The commands are all run with errexit and pipefail, which will at least fail the build if something goes sideways. We don't yet have actual tests to run within a completed vagrant box.